### ISSUE TITLE
networking leak nsfs  mounts

### ISSUE THREAD

**Issue by baude** [projectatomic/libpod#163]
when using networking, we are leaking/not cleaning up nsfs mounts after the container stops.

for example, after running:
```
bin/podman --log-level=debug run -it --rm --network=bridge alpine ip a
```
a newly created mount of:
```
nsfs on /run/netns/cni-94a7ea7b-ebad-cff0-56d2-96d6be27db5c type nsfs (rw,seclabel)
```
remains after the container is done executing and removed.

**Comment by mheon** [comment]
It remains after a `podman rm`? That seems pretty serious, as the routines to destroy namespaces should be firing then.

**Comment by baude** [comment]
yes it remains ... i also sent you an email about several other things ... take a peek

**Comment by mheon** [comment]
Fixed in #197

**Comment by rhatdan** [comment]
Please open a new issue. Almost guaranteed you have a different problem.
